06_gpu_and_ml/vision_model_training.py

```diff
@@ -290,20 +292,24 @@ def fastapi_app():
     )
 
 
-if __name__ == "__main__":
-    cmd = sys.argv[1] if len(sys.argv) >= 2 else "train"
-    if cmd == "classify":
-        image_url = sys.argv[2]
-        with stub.run():
-            classify_url.call(image_url)
-    elif cmd == "train":
-        with stub.run():
-            train.call()
-    elif cmd == "serve":
-        stub.serve()
-    elif cmd == "shell":
-        stub.interactive_shell()
-    else:
-        print(f"Invalid cmd '{cmd}'.")
-
+## Running this
+#
+# To run training as an ephemeral app:
+#
+# ```shell
+# modal run vision_model_training.py::stub.train
+# ```
+#
+# To test the model on an image, run:
+#
+# ```shell
+# modal run vision_model_training.py::stub.classify_url --image-url <url>
```

Comment:
Do we have any ideas about how to make these ref less complex?

Reply:
@freider yeah maybe we could make it a bit more user friendly?